bears/general/AnnotationBear.py

```diff
@@ -44,9 +48,9 @@ def run(self, filename, file, language: str, coalang_dir: str = None):
         multiline_comment_delimiters = dict(
             lang_dict['multiline_comment_delimiters'])
         comment_delimiter = dict(lang_dict['comment_delimiter'])
-        string_ranges = comment_ranges = ()
+        ranges = ((), (), (), ())
```

Comment:
So, I had a thought of creating a `RangeHelper` (like the rangehelper module here https://github.com/coala/coala-bears/pull/1528/files). So are you talking specifically for this module or in general ?